source/base/parameter_handler.cc

```diff
@@ -313,31 +318,38 @@ namespace Patterns
 
   Double *Double::create (const std::string &description)
   {
-    if (description.compare(0, std::strlen(description_init), description_init) == 0)
-      {
-        std::istringstream is(description);
+    const std::string description_init_str = description_init;
+    if (description.compare(0, description_init_str.size(), description_init_str) != 0)
+      return NULL;
+    if (description.back() != ']')
+      return NULL;
 
-        if (is.str().size() > strlen(description_init) + 1)
-          {
-            double lower_bound, upper_bound;
+    std::string temp = description.substr(description_init_str.size());
+    if (temp == "]")
+      return new Double(1.0, -1.0); // return an invalid range
```

Comment:
This invalid range functionality does not seem to be documented. Could you add something to the header file to describe this behavior?